index.js

```diff
@@ -1,9 +1,7 @@
 /**
  * Map of HTML elements to allowed attributes.
- *
- * @type {Record<string, Array<string>>}
  */
-export const htmlElementAttributes = {
+export const htmlElementAttributes = /** @type {const} */ ({
```

Comment:
This file wasn't actually regenerated, just manually edited. I did run the generator but it also spat out a bunch of other changes that I thought you'd want to review separately.